content/contact/index.md

```diff
@@ -1,6 +1,5 @@
 ---
 title: Contact
-summary: "GraphWalker is an open source Model-based testing tool for test automation. This page contains information when you need to ask questions or to get support on GraphWalker."
```

Comment:
No, not other than it was present on some pages and not others, and was slightly changed on different pages. If you click around on the site, you see it very often. I thought it might suffice to just have it on the front page.